app/src/main/java/kr/co/nottodo/presentation/toplevel/recommendation/recommendationfragment/RecommendationFragment.kt

```diff
@@ -0,0 +1,101 @@
+package kr.co.nottodo.presentation.toplevel.recommendation.recommendationfragment
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import kr.co.nottodo.R
+import kr.co.nottodo.databinding.FragmentRecommendationBinding
+import kr.co.nottodo.presentation.schedule.addition.view.AdditionActivity
+import kr.co.nottodo.presentation.toplevel.recommendation.data.responsedto.ResponseRecommendationCategorySituationDto
+import kr.co.nottodo.presentation.toplevel.recommendation.recommendationlist.RecommendationParentAdapter
+import kr.co.nottodo.presentation.toplevel.recommendation.viewmodel.RecommendationViewModel
+import timber.log.Timber
+
+
+class RecommendationFragment : Fragment() {
+
+    private var _binding: FragmentRecommendationBinding? = null
+    private val binding: FragmentRecommendationBinding
+        get() = requireNotNull(_binding) { "_binding is Null ..." }
+    private lateinit var recommendationAdapter: RecommendationAdapter
+    private lateinit var parentAdapter: RecommendationParentAdapter
+    private val viewModel by viewModels<RecommendationViewModel>()
+    private var itemList: List<ResponseRecommendationCategorySituationDto.CategorySituation> =
+        listOf()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = DataBindingUtil.inflate<FragmentRecommendationBinding>(
+            inflater, R.layout.fragment_recommendation, container, false
+        )
+        return binding.root
+
+    }
+
+    val sampleList = listOf(
+        "SNS",
+        "작업,업무",
+        "건강",
+        "생활습관",
+        "기상,취침",
+    )
```

Comment:
이친구는 companion object에다가 적으면 좋을 것 같아(상수니까)

```suggestion
    companion object {
        val sampleList = listOf(
            "SNS",
            "작업,업무",
            "건강",
            "생활습관",
            "기상,취침",
        )    
}
```